src/azul/__init__.py

```diff
@@ -1432,7 +1432,7 @@ def docker_registry(self) -> str:
     # the `shared` TF component.
 
     docker_images = [
-        'docker.io/ucscgi/azul-elasticsearch:7.17.10-2',
+        'docker.io/ucscgi/azul-elasticsearch:7.17.10-3',
```

Comment:
Please ask @achave11-ucsc to bump the internal version (he can push to the `main` branch now, without a PR) and then 

```suggestion
        'docker.io/ucscgi/azul-elasticsearch:7.17.10-4,
```

Reply:
I'm unable to push to the `main` branch. I get the following error:
```
❯ git push github main
Enumerating objects: 9, done.
Counting objects: 100% (9/9), done.
Delta compression using up to 10 threads
Compressing objects: 100% (3/3), done.
Writing objects: 100% (5/5), 465 bytes | 465.00 KiB/s, done.
Total 5 (delta 1), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (1/1), completed with 1 local object.
remote: error: GH006: Protected branch update failed for refs/heads/main.
remote: error: Required status check "build" is expected.
To https://github.com/DataBiosphere/azul-docker-elasticsearch.git
 ! [remote rejected] main -> main (protected branch hook declined)
error: failed to push some refs to 'https://github.com/DataBiosphere/azul-docker-elasticsearch.git'
```

Should I just push the change on a separated branch, get the build check, and then commit on `main`?